### ISSUE TITLE
Custom "glamor" rules not overriding default

### ISSUE THREAD

**Issue by ifier** [fkhadra/react-toastify#90]
Hi. I'm trying to customize toastify and I have some issues with that:
1) http://prntscr.com/hj9gla
2) http://prntscr.com/hj9gu2
3) http://prntscr.com/hj9gxq

It's adding new class name but it didn't added new attribute to the div :( That's why I can't override default styles. Maybe I'm doing something wrong? But I've read "glamor" documentation and your as well and it seems like that all okay with my code. Can you help?

Also I think it would be awesome (like new feature) - disable glamor if someone don'r wanna it.

**Comment by fkhadra** [comment]
Hello @ifier,

I did the following:
```js
<ToastContainer toastClassName={
          css({
            background: "#3f4650"
          })
        }
        progressClassName={css({
          height: "2px"
        })}
        />
```
And I got the expected result. Maybe you are overidding the style somewhere else ? Could you provide a snippet so I can help you better ?
![toastify-issue-90](https://user-images.githubusercontent.com/5574267/33596961-1695811c-d99e-11e7-9e10-1feddcb875fa.gif)

Regarding your last request, I'll see if there is a good way to choose a style strategy(glamor or css)

**Comment by fkhadra** [comment]
@ifier when I look your screenshot, you are overriding the background color with the same value.
![8lmieymyqr_okvytbqa92w](https://user-images.githubusercontent.com/5574267/33597259-1ef9acba-d99f-11e7-931b-90d5e86189f1.png)

**Comment by ifier** [comment]
Okay. I've made with CSS class names and `!important` (which is not great) for now.
I've notices strange behavior with glamor - it works not well with hot reload :(

I've made like I wrote before and refresh a page - styles are not working. But when I changed some styles and `hot-reload` refreshed `component/page` it applied new styles that I wrote...

Also `glamor` is heavy-weight module :( Maybe I'm old school guy and I think css must be in css files :)

P.S.: hot reload that we are using - webpack-dev-middleware / webpack-hot-middleware
